app/src/main/java/org/wikipedia/talk/TalkReplyActivity.kt

```diff
@@ -66,7 +70,8 @@ class TalkReplyActivity : BaseActivity(), LinkPreviewDialog.Callback, UserMentio
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         title = ""
 
-        linkHandler = TalkLinkHandler(this)
+        invokeSource = intent.getSerializableExtra(INTENT_EXTRA_INVOKE_SOURCE) as Constants.InvokeSource
```

Comment:
Can we use `viewModel.isNewTopic` here instead of having the `invokeSource` here?